src/com/projectkorra/projectkorra/airbending/Suffocate.java

```diff
@@ -149,7 +150,12 @@ public Suffocate(final Player player) {
 	public void progress() {
 		for (int i = 0; i < this.targets.size(); i++) {
 			final LivingEntity target = this.targets.get(i);
-			if (target.isDead() || !target.getWorld().equals(this.player.getWorld()) || target.getLocation().distanceSquared(this.player.getEyeLocation()) > this.range * this.range || GeneralMethods.isRegionProtectedFromBuild(this, target.getLocation())) {
+
+			if (target.isDead() ||

```

Comment:
While I understand why you would change the formatting here, in order to keep it uniform with the rest of the plugin could you please change it back...

Reply:
ok